lumi/lumi_jobscript.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
-#SBATCH --output=/scratch/project_465001383/%u/logs/%x.out
-#SBATCH --error=/scratch/project_465001383/%u/logs/%x.err
+#SBATCH --output=/scratch/project_465001383/%u/logs/%x_%j.out
+#SBATCH --error=/scratch/project_465001383/%u/logs/%x_%j.err
 #SBATCH --nodes=1
```

Comment:
This is a good change, but can we make it the user folder instead? `/users/%u/logs/%x_%j` (not everyone have named their project user folder after their username)

Reply:
Done